molecule-runtime/src/darwinTest/kotlin/app/cash/molecule/DisplayLinkClockTest.kt

```diff
@@ -15,14 +15,20 @@
  */
 package app.cash.molecule
 
+import assertk.all
+import assertk.assertThat
+import assertk.assertions.isLessThan
+import assertk.assertions.isPositive
 import kotlin.test.Test
 import kotlinx.coroutines.test.runTest
 
 class DisplayLinkClockTest {
-
-  @Test fun `DisplayLinkClock delivers a single frame`() = runTest {
-    DisplayLinkClock.withFrameNanos {
-      // If this function does not time out the test passes.
+  @Test fun ticksWithTime() = runTest {
+    val frameTimeA = DisplayLinkClock.withFrameNanos { it }
+    val frameTimeB = DisplayLinkClock.withFrameNanos { it }
+    assertThat(frameTimeA).all {
+      isPositive()
+      isLessThan(frameTimeB)
```

Comment:
Is it testing that `frameTimeA` ran before `frameTimeB`? and we assume that the code is executed fast enough for it to make sense?

Reply:
Yes. Because it's nanos it's basically impossible to get two of the same values back since the precision is so high.